STYLEGUIDE.md

```diff
@@ -322,6 +325,99 @@ function changeValues(object) {
 
 ---
 
+## <a id="Forwarding-Refs">Forwarding refs</a>
+
+Ref forwarding allows us to provide direct access to the underlying parent element. For more information on ref forwarding, check out the React [docs](https://reactjs.org/docs/forwarding-refs.html).
```

Comment:
nit, header should be
`#### Why`

Reply:
Yeah the id should be added in automatically by github

Reply:
@spark33 how does that work?